m2cgen/interpreters/mixins.py

```diff
@@ -195,3 +195,46 @@ def _get_subroutine_name(self):
 
     def create_code_generator(self):
         raise NotImplementedError
+
+
+class InfixPowExprMixin(BaseToCodeInterpreter):
```

Comment:
IMHO, `PowExprInfixMixin` and `PowExprFunctionMixin` names are slightly better because the first word highlights a context about what these classes are (they are about for to implement power expression).

Also, it would be easier for contributors to search for appropriate mixin for a new programming language.

Just as an example: https://docs.opencv.org/3.4/d8/d01/group__imgproc__color__conversions.html

Note enumerations are in a form of `COLOR_xxx` but not in a form of `xxx_COLOR`.